husarion_ugv_utils/test/test_moving_average.cpp

```diff
@@ -61,12 +61,12 @@ TEST(TestMovingAverage, TestHighOverload)
 
   double sum = 0.0;
   for (std::size_t i = 1; i <= window_len * 10; i++) {
-    sum += double(i);
+    sum += double(i) / double(window_len);
     ma.Roll(double(i));
 
     // test every 1000 rolls expected average
     if (i % window_len == 0) {
-      EXPECT_EQ(sum / double(window_len), ma.GetAverage());
+      EXPECT_LT(sum - ma.GetAverage(), std::numeric_limits<double>::epsilon());
```

Comment:
Please use `ASSERT_NEAR(val1, val2, abs_error);` function if you'd like to compare two values with some treshold.